modules/structs/issue.go

```diff
@@ -170,7 +170,7 @@ func (it IssueTemplate) Type() IssueTemplateType {
 	if ext := filepath.Ext(it.FileName); ext == ".md" {
 		return IssueTemplateTypeMarkdown
 	} else if ext == ".yaml" || ext == ".yml" {
-		return "yaml"
+		return IssueTemplateTypeYaml
 	}
-	return IssueTemplateTypeYaml
+	return ""
```

Comment:
I haven't tested whether the newly returned `""` works (it might change some behaviors). I think it should be OK. So LGTM and let's try.

Reply:
😂 Don't worry, it should be `return ""`, I wrote the wrong code with a drunk brain.